Figure readme updates (#24)
greenelab/annorxiver: biorxiv/README.md

# Directory Structure

> | Folder/file | Description |

Reviewer: Based on this description it doesn't seem like there is an order to running the notebooks in these different folders. Is that correct. If not, you might want to specify this somewhere

Author: >Based on this description it doesn't seem like there is an order to running the notebooks in these different folders. Is that correct. If not, you might want to specify this somewhere 

That is correct. There isn't a specified order to run notebooks.

| --- | --- | 

> | [article_distances](article_distances) | This folder contains notebooks that aimed to find missing links within bioRxiv. |

Reviewer 2: ```suggestion
| [article_distances](article_distances) | This folder contains notebooks that aim to find missing links within bioRxiv. |
```

> | [corpora_comparison ](corpora_comparison ) | This folder contains notebooks that aimed to compare bioRxiv with PMC and NYTAC. | 

Reviewer 2: ```suggestion
| [corpora_comparison ](corpora_comparison ) | This folder contains notebooks that aim to compare bioRxiv with PMC and NYTAC. | 
```

> | [exploratory_data_analysis](exploratory_data_analysis) | This folder contains notebooks that obtains metadata from the bioRxiv respository. |

Reviewer: ```suggestion
| [exploratory_data_analysis](exploratory_data_analysis) | This folder contains notebooks that obtain metadata from the bioRxiv respository. |
```

> | [journal_tracker](journal_tracker) | This folder contains code map preprints to their published counterparts. |

Reviewer: ```suggestion
| [journal_tracker](journal_tracker) | This folder contains code that maps preprints to their published counterparts. |
```

> | [missing_sections_experiment](missing_sections_experiment) | This folder contains notebooks that aimed to find if a preprint is missing a section. |

Reviewer 2: ```suggestion
| [missing_sections_experiment](missing_sections_experiment) | This folder contains notebooks that aim to find if a preprint is missing a section. |
```
